x/group/client/testutil/cli_test.go

```diff
@@ -4,15 +4,19 @@
 package testutil
 
 import (
+	_ "embed"
 	"testing"
 
 	"github.com/cosmos/cosmos-sdk/testutil/network"
+	"github.com/cosmos/cosmos-sdk/x/group/testutil"
 
+	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 )
 
 func TestIntegrationTestSuite(t *testing.T) {
-	cfg := network.DefaultConfig()
+	cfg, err := network.DefaultConfigWithAppConfig(testutil.AppConfig)
+	require.NoError(t, err)
 	cfg.NumValidators = 2
```

Comment:
group tests are failing with panic `subspace already occupied` when no. of validators in config is > 1. 
https://github.com/cosmos/cosmos-sdk/runs/6874035724?check_suite_focus=true
How should we proceed here?

Reply:
Should be fixed on main